lldb/source/Plugins/TypeSystem/Swift/SwiftASTContext.cpp

```diff
@@ -8303,6 +8303,18 @@ bool SwiftASTContext::GetImplicitImports(
   return true;
 }
 
+void SwiftASTContext::LoadStringProcessingModule(TargetSP target,
+                                                 ProcessSP process) {
+  static std::once_flag once_flag;
```

Comment:
I think the once flag need to be owned by SwiftASTContext and not global. lldb-rpc-server is very long-lived.